main/lsp/requests/document_formatting.cc

```diff
@@ -53,6 +53,18 @@ void DocumentFormattingTask::displayError(string errorMessage, unique_ptr<Respon
         "2.0", LSPMethod::WindowShowMessage, make_unique<ShowMessageParams>(MessageType::Error, errorMessage)));
 }
 
+bool DocumentFormattingTask::documentIsFormattable(string_view path) {
+    if (config.opts.rubyfmtFormattableDirectories.empty()) {
+        return true;
+    }
+    for (string directory : config.opts.rubyfmtFormattableDirectories) {
+        if (path.find(directory) == 0 || path.find(fmt::format("./{}", directory)) == 0) {
```

Comment:
You almost never want to hardcode `./` in Sorbet, as the prefix in front of the file path will be whatever was passed to the `--dir` flag.

Instead, always use `config.opts.pathPrefix`.